nextflow.config

```diff
@@ -11,14 +11,15 @@ params {
   // Pipeline Options
   project = false
   aligner = 'star'
+  gene_counter = 'htseq'
```

Comment:
As the default was `featureCounts` for everyone until now, I'm a bit hesitant to change the default to HTSeq now (as it will "silently" give different results for users that were using V1.0-V1.3 of this pipeline).

Can we have featureCounts as default? Having the option now is super nice and we can advertise that in the upcoming changelog for V1.4!

Reply:
I agree :+1: